### ISSUE TITLE
Wrong nVidia drivers offered by default

### ISSUE THREAD

**Issue by Willdrick** [solus-project/doflicky#1]
TL;DR DoFlicky should show at least the 340 version for the GeForce 650ti cards. Currently only offering 304.

After a fresh install I went to the software centre and updated everything. Next reboot I opened up DoFlicky to install the nVidia drivers but it was only showing the 304.131 version, which is much older than the >340 I use to install. After installation I rebooted and found that my mouse clicks weren't registering. I could see the cursor and move it but not click.
Installed 340 release through eopkg and solved the driver issue.

extra info:
Linux shodan 4.4.14 #1 SMP PREEMPT Sat Jun 25 21:14:21 UTC 2016 x86_64 GNU/Linux
01:00.0 VGA compatible controller: NVIDIA Corporation Device 11c6 (rev a1)
[dmesg.txt](https://github.com/solus-project/doflicky/files/333940/dmesg.txt)
[lspci.txt](https://github.com/solus-project/doflicky/files/333939/lspci.txt)

**Comment by dhruvmittal** [comment]
Had identical issue and solution with Nvidia Quadro 880M card.

**Comment by ikeydoherty** [comment]
yeah driver stuff is being fixed up to be less derpy.. i.e. avoid these detection issues
